### ISSUE TITLE
[Feature] Add support for 'cut numbers' in generated Morse

### ISSUE THREAD

**Issue by pwdirks** [MorseKOB/PyKOB#66]
"Cut Numbers" are letters used in place of common numbers.  See https://wb8ylo.wordpress.com/2014/12/04/cw-shorthand-cut-numbers/ for a table of abbreviations for otherwise 5-symbol number representations and their "cut number" variants:

Number         Normal Code           Cut Number         Result
  1        di dah dah dah dah      di dah                  A
  2        di di dah dah dah       di di dah               U
  3        di di di dah dah        di di di dah            V
  4        di di di di dah         di di di di dah         4
  5        di di di di dit         dit                     E
  6        dah di di di dit        dah di di di dit        6
  7        dah dah di di dit       dah dah dit             G
  8        dah dah dah di dit      dah di dit              D
  9        dah dah dah dah dit     dah dit                 N
  0        dah dah dah dah dah     dah                     T

It would be cool if either the core Morse encoding engine or specific applications like Clock.py could add optional support for using "cut numbers" to represent the numeric content in the message, perhaps in various levels (e.g. only use 'N' for '9', or also use 'T' for '0', or use A,U,V,E,G,and D as well?)

**Comment by leskerr** [comment]
> I was wondering whether it was a ham-only thing but, given the volume of numeric info to be conveyed (times, train #s, etc.) I'd be surprised if there wasn't some shortcut used - at least using "O" ('Oh") for "0" (zero)?

Yes, definitely the letter O (dit dit) instead of the official code for zero (long dash, daaah) was universally used in American Morse. So much so that for PyKOB's code sender I departed from MorseKOB 2.5 and decided it should send two dots for the zero.

I do know that cut numbers weren't used by railroad telegraphers. Other than the zero, for numerals they used the official codes (which are shorter than they are in International Code) for everything -- times, engine numbers, car numbers, etc.

The open question is what was used on landline press wires. Those wires experienced extremely heavy traffic and made extensive use of abbreviations (e.g., the [Phillips code](https://en.wikipedia.org/wiki/Phillips_Code)). If we're going to find use of cut numbers (or something similar) in American Morse, that's where it would be. Also possibly stock reports.

**Comment by pwdirks** [comment]
Sounds like it may be an option applicable only if international morse is being generated.

**Comment by leskerr** [comment]
International Morse may be the only situation where it's used, but if it's simpler to implement then there's probably no harm in making the option available in both codes and letting the user decide whether to turn it on or not. (There's my 'lazy gene' expressing itself again!)
